### PR TITLE
Ec download templates

### PR DESCRIPTION
**Pull request by tommyd450**

### CODE REVIEW

File: charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{{- if .Values.configs.clientserver.consoleDownload }}
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: enterprisecontract
+spec:
+  description: A copy of EnterpriseContract
```
Comment (simonbaird) [inline charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml:7]: Maybe something more user friendly here, e.g.:

```suggestion
  description: Enterprise Contract is a tool for verifying signatures and performing policy checks on container images

File: charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  displayName: EnterpriseContract - Command Line Interface (CLI)
```
Comment (simonbaird) [inline charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml:8]:
```suggestion
  displayName: Enterprise Contract - Command Line Interface (CLI)
```

**Review by simonbaird — COMMENTED** [review]
Some name and description nit picks, otherwise lgtm

File: charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml (hunk 2 above)
Comment (simonbaird) [reply charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml:8]: We spell it with a space.

Comment (tommyd450) [reply charts/trusted-artifact-signer/templates/consoleclidownload-ec.yaml:8]: Awesome, Ill add these changes now.

**Pull request merged**